ReactiveCocoa.playground/Pages/Signal.xcplaygroundpage/Contents.swift

```diff
@@ -1,11 +1,15 @@
 /*:
  > # IMPORTANT: To use `ReactiveCocoa.playground`, please:
  
- 1. Open `ReactiveCocoa.xcworkspace`
```

Comment:
@iv-mexx yeah I didn't quite understand it myself at first either. 3066cf2 brings the playground instructions in the main [README](https://github.com/ReactiveCocoa/ReactiveCocoa/blob/master/README.md) up to date with the new playground steps